### ISSUE TITLE
TypeError: Cannot read property 'getCharacteristic' of undefined

### ISSUE THREAD

**Issue by pbr90x** [nzapponi/homebridge-simplisafe3#71]
I'm thinking this issue might be related to the recent changes with get and set functions. With the most recent release I started seeing an error on launch. It doesn't seem to be fatal, but wanted to submit this up.

```
Feb 01 12:41:03 raspberrypi homebridge[3505]: [2/1/2020, 12:41:03] [Home Alarm] SS3 refresh failed
Feb 01 12:41:03 raspberrypi homebridge[3505]: [2/1/2020, 12:41:03] [Home Alarm] TypeError: Cannot read property 'getCharacteristic' of undefined
Feb 01 12:41:03 raspberrypi homebridge[3505]:     at SS3SimpliCam.setAccessory (/usr/lib/node_modules/homebridge-simplisafe3/accessories/simplicam.js:64:76)
Feb 01 12:41:03 raspberrypi homebridge[3505]:     at /usr/lib/node_modules/homebridge-simplisafe3/index.js:128:18
Feb 01 12:41:03 raspberrypi homebridge[3505]:     at processTicksAndRejections (internal/process/task_queues.js:94:5
```

I'm running Raspbian Buster, with the Node.js package installed from the Node.js apt repository targeting Debian Buster:

```
pi@raspberrypi:/home $ cat /etc/apt/sources.list.d/nodesource.list
deb https://deb.nodesource.com/node_12.x buster main
deb-src https://deb.nodesource.com/node_12.x buster main
```

```
pi@raspberrypi:/home $ dpkg -l nodejs
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version              Architecture Description
+++-==============-====================-============-===========================
ii  nodejs         12.14.1-1nodesource1 armhf        Node.js event-based server-
lines 1-6/6 (END)
```

**Comment by nikonratm** [comment]
Those changes haven't been merged yet but there were some changes in #60 that may be related. What type of camera are you having issues with (a doorbell or no)? Have you tried clearing the persist / cachedAccessories folders (reminder: this may remove your accessories so that can be a pain)?

**Comment by pbr90x** [comment]
As suggested, I stopped homebridge, blew away my persist and cachedAccessories directories and started homebridge again. After removing homebridge as a HomeKit bridge from my home, re-adding it, and re-adding my accessories and my two SimpliSafe cameras (SimpliCam, Doorbell) in the iOS Home app, things seem to be working smoothly.
